fast/stages/02-networking-nva/data/subnets/dev/dev-default-ew1.yaml

```diff
@@ -1,5 +1,5 @@
 # skip boilerplate check
 
 region: europe-west1
-ip_cidr_range: 10.128.128.0/19
+ip_cidr_range: 10.128.128.0/20
```

Comment:
Per my comment, individual subnets are by default /24 carved from the IP space assigned to a specific spoke/region.